app/views/users/_list.haml

```diff
@@ -0,0 +1,11 @@
+%tbody
+  - @users.each do |user|
```

Comment:
Can we pass in `users` as a local variable instead of using `@users`? Would be nice to start relying on strict locals more for our partials!

Reply:
Would kind of prefer not to do that in this PR? Since it's a separate topic

Reply:
Hm, ok. Clean-up PR next? :D

Reply:
-> handled